virttest/utils_windows/system.py

```diff
@@ -58,3 +58,17 @@ def file_exists(session, filename):
     if not session.cmd_status(check_cmd):
         return True
     return False
+
+
+def escape_cmd_string(string):
```

Comment:
The name of this function is quit misleading.
I don't have a good name for it, based on your doc sting, maybe:
`escape_cmd_symbols(win_cmd)`,  just a reference for you.
You may come up a more descriptive func and param name.